assignments/objects.js

```diff
@@ -19,27 +21,71 @@ const example = {
 
 // Write your intern objects here:
 
+const interns = {
+  mitzi : {
+    id      : 1,
+    email    : 'mmelloy0@psu.edu',
+    name   : 'Mitzi',
+    gender  : 'F'
+  },
+  kennan : {
+    id      : 2,
+    email    : 'kdiben1@tinypic.com',
+    name   : 'Kennan',
+    gender  : 'M'
+  },
+  keven : {
+    id      : 3,
+    email    : 'kmummery2@wikimedia.org',
+    name   : 'Keven',
+    gender  : 'M'
+  },
+  gannie : {
+    id      : 4,
+    email    : 'gmartinson3@illinois.edu',
+    name   : 'Gannie',
+    gender  : 'M'
+  },
+  antonietta : {
+    id      : 5,
+    email    : 'adaine5@samsung.com',
+    name   : 'Antonietta',
+    gender  : 'F'
+  }
+}
```

Comment:
great way to organize your interns. it is more traditional to use an array for this but your approach is also quite useful and concise.